tests/functional/model/run_model_test.py

```diff
@@ -94,3 +107,58 @@ def test_llm_run_with_file():
     # Verify response
     assert response["status"] == "SUCCESS"
     assert "🤖" in response["data"], "Robot emoji should be present in the response"
+
+
+def test_index_model_with_image():
+    from aixplain.factories import IndexFactory
+    from aixplain.modules.model.record import Record
+    from uuid import uuid4
+
+    for index in IndexFactory.list()["results"]:
+        index.delete()
+
+    index_model = IndexFactory.create(
+        name=f"Image Index {uuid4()}", description="Index for images", embedding_model=EmbeddingModel.JINA_CLIP_V2_MULTIMODAL
```

Comment:
If this model works only for image input, should we check the type if this embedding was selected?